mlflow/types/llm.py

```diff
@@ -701,7 +700,8 @@ class ChatCompletionChunk(_BaseDataclass):
         object (str): The object type. Defaults to 'chat.completion.chunk'
         created (int): The time the response was created.
             **Optional**, defaults to the current time.
-        custom_outputs (Dict[str, str]): An field that can contain arbitrary additional context.
+        custom_outputs (Dict[str, any]): An field that can contain arbitrary additional context.
```

Comment:
Looks like the docs are failing to build for this (seeing an error like `py:class reference target not found: any`), maybe it needs to be `Any`?

```suggestion
        custom_outputs (Dict[str, Any]): An field that can contain arbitrary additional context.
```